_episodes/01-run-quit.md

```diff
@@ -102,37 +250,38 @@ keypoints:
 > >
 > > Command mode has a grey border and Edit mode has a green border. 
 > > Use <kbd>Esc</kbd> and <kbd>Return</kbd> to switch between modes. 
-> > You need to be in Command mode (Press <kbd>Esc</kbd> if your cell is green).  Type <kbd>B</kbd> or <kbd>A</kbd>.
-> > You need to be in Command mode (Press <kbd>Esc</kbd> if your cell is green).  Type <kbd>X</kbd>.
+> > You need to be in Command mode (Press <kbd>Esc</kbd> if your cell is blue).  Type <kbd>B</kbd> or <kbd>A</kbd>.
+> > You need to be in Command mode (Press <kbd>Esc</kbd> if your cell is blue).  Type <kbd>X</kbd>.
 > >
 > {: .solution}
 {: .challenge}
 
-## Use the keyboard and mouse to select and edit cells.
+### Use the keyboard and mouse to select and edit cells.
 
-*   Pressing the <kbd>Return</kbd> key turns the border green and engages Edit mode,
-    which allows you to type within the cell.
+*   Pressing the <kbd>Return</kbd> key turns the border blue and engages Edit mode, which allows 
+    you to type within the cell.
 *   Because we want to be able to write many lines of code in a single cell,
-    pressing the <kbd>Return</kbd> key when in Edit mode (green) moves the cursor to the next line in the cell just like in a text editor.
+    pressing the <kbd>Return</kbd> key when in Edit mode (blue) moves the cursor to the next line 
+    in the cell just like in a text editor.
 *   We need some other way to tell the Notebook we want to run what's in the cell.
 *   Pressing <kbd>Shift</kbd>+<kbd>Return</kbd> together will execute the contents of the cell.
-*   Notice that the <kbd>Return</kbd> and <kbd>Shift</kbd> keys on the 
-    right of the keyboard are right next to each other.
+*   Notice that the <kbd>Return</kbd> and <kbd>Shift</kbd> keys on the right of the keyboard are 
+    right next to each other.
 
-## The Notebook will turn Markdown into pretty-printed documentation.
+### The Notebook will turn Markdown into pretty-printed documentation.
 
 *   Notebooks can also render [Markdown][markdown].
     *   A simple plain-text format for writing lists, links, 
         and other things that might go into a web page.
     *   Equivalently, a subset of HTML that looks like what you'd send in an old-fashioned email.
-*   Turn the current cell into a Markdown cell by entering 
-    the Command mode (<kbd>Esc</kbd>/gray) and press the <kbd>M</kbd> key.
-*   `In [ ]:` will disappear to show it is no longer a code cell
-    and you will be able to write in Markdown.
-*   Turn the current cell into a Code cell
-    by entering the Command mode (<kbd>Esc</kbd>/gray) and press the <kbd>Y</kbd> key.
+*   Turn the current cell into a Markdown cell by entering the Command mode (<kbd>Esc</kbd>/gray) 
+    and press the <kbd>M</kbd> key.
+*   `In [ ]:` will disappear to show it is no longer a code cell and you will be able to write in 
```

Comment:
I believe the `In` part may be gone depending in JupyterLab version, just the `[ ]:` may stay.